src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -18,20 +18,55 @@ import {
 
 import { CSP_IDS } from '~/platform/user/authentication/constants';
 import { signInServiceName } from '~/platform/user/authentication/selectors';
-import { isLOA1 as isLOA1Selector } from '~/platform/user/selectors';
+import { selectAuthStatus } from '../../selectors/auth-status';
 import {
-  notFoundComponent,
-  unavailableComponent,
   downloadErrorComponent,
   errorTypes,
+  notFoundComponent,
+  systemErrorComponent,
+  unavailableComponent,
 } from './utils';
 import '../../sass/download-1095b.scss';
 
-export const App = ({ displayToggle, isLOA1, loggedIn, toggleLoginModal }) => {
+export const App = ({ displayToggle, toggleLoginModal }) => {
   const [year, updateYear] = useState(0);
   const [formError, updateFormError] = useState({ error: false, type: '' });
   const cspId = useSelector(signInServiceName);
   const [verifyAlertVariant, setverifyAlertVariant] = useState(null);
+  const [availableForms, setAvailableForms] = useState(null);
+  const profile = useSelector(state => selectAuthStatus(state));
+  const isAppLoading = useMemo(
+    () => {
+      return profile.isLoadingProfile || availableForms === null;
+    },
+    [profile, availableForms],
+  );
+
+  useEffect(() => {
+    const getAvailableForms = () => {
+      return apiRequest('/form1095_bs/available_forms')
+        .then(response => {
+          if (response.errors || response.availableForms.length === 0) {
+            updateFormError({ error: true, type: errorTypes.NOT_FOUND });
+          }
+          return response.availableForms === null
+            ? false
+            : response.availableForms;
+        })
+        .catch(() => {
+          updateFormError({ error: true, type: errorTypes.SYSTEM_ERROR });
+          return false; // Return false in case of an error
+        });
+    };
+
+    getAvailableForms().then(forms => {
+      setAvailableForms(forms);
+      const mostRecentYearData = forms[0];
+      if (mostRecentYearData?.lastUpdated && mostRecentYearData?.year) {
```

Comment:
I think we can remove `mostRecentYearData?.lastUpdated` because we don't actually use lastUpdated anymore.

Reply:
Removed.